ggshield/cmd/sca/scan/__init__.py

```diff
@@ -0,0 +1,19 @@
+from typing import Any
+
+import click
+
+from ggshield.cmd.sca.scan.ci import ci_cmd
+from ggshield.cmd.sca.scan.diff import scan_pre_commit_cmd
+from ggshield.cmd.sca.scan.scan import scan_all_cmd
+
+
+@click.group(
```

Comment:
nit: it looks a little bit strange to me to define that in the `__init__` file, I would have let that in the `scan.py` file

Reply:
Yep I agree with renaming the file that contains `scan_all`. Maybe @amascia-gg will have an opinion on this.

Reply:
We should follow what is done for IaC and secrets so I'm in favour for Jonathan's implementation